data/tribes/scripting/help/controls.lua

```diff
@@ -67,6 +75,12 @@ return {
                -- TRANSLATORS: This is an access key combination. The hotkey is 'e'
                dl(help_format_hotkey("E"), _"Toggle seafaring statistics") ..
                -- TRANSLATORS: This is an access key combination. Localize, but do not change the key.
+               dl(help_format_hotkey(pgettext("hotkey", "Arrow keys")), _"Map view movement") ..
```

Comment:
I'd prefer "Move the map fast/slowly etc, because we're using verbs for the other entries.